examples/common/c/configfile.c

```diff
@@ -659,21 +659,24 @@ void config_setstd(struct RastaConfig *cfg) {
     // TLS settings
 
     entr = config_get(cfg, "RASTA_CA_PATH");
+    cfg->values.tls.ca_cert_path = NULL;
     if (entr.type == DICTIONARY_STRING) {
-        cfg->values.tls.ca_cert_path[PATH_MAX - 1] = 0;
-        memcpy(cfg->values.tls.ca_cert_path, entr.value.string.c, PATH_MAX);
+        cfg->values.tls.ca_cert_path = malloc(MAX_DICTIONARY_STRING_LENGTH_BYTES);
+        strcpy(cfg->values.tls.ca_cert_path, entr.value.string.c);
```

Comment:
`strcpy` does not check buffer lengths, which might become dangerous when reading unknown input. We should prefer `strncpy` or something similar.

Reply:
Thanks for the comments! Should be fixed now, please re-review.